tests/spec/core/xref-spec.js

```diff
@@ -675,23 +676,29 @@ describe("Core — xref", () => {
       const ops = makeStandardOps(config, body);
       const doc = await makeRSDoc(ops);
 
-      const [link1a, link1b] = [...doc.querySelectorAll("#link1 code a")];
+      const [link1a, link1b] = [...doc.querySelectorAll("#link1 a")];
       expect(link1a.href).toBe(expectedLinks.get("EventTarget"));
       expect(link1b.href).toBe(
         expectedLinks.get("EventTarget.addEventListener")
       );
+      expect(link1a.firstElementChild.localName).toBe("code");
+      expect(link1b.firstElementChild.localName).toBe("code");
       const vars1 = [...doc.querySelectorAll("#link1 var")];
       expect(vars1.length).toBe(2);
       expect(vars1[0].textContent).toBe("type");
       expect(vars1[1].textContent).toBe("callback");
 
-      const [link2a, link2b] = [...doc.querySelectorAll("#link2 code a")];
+      const [link2a, link2b] = [...doc.querySelectorAll("#link2 a")];
       expect(link2a.href).toBe(expectedLinks.get("ChildNode"));
+      expect(link2a.firstElementChild.localName).toBe("code");
```

Comment:
so these tests (localName ones) are all kinda same.. maybe do them once per type?

Reply:
I wanted to keep the original structure we were testing, which structurally was "#link code a" for all of them (now just swapping a and code). I know it's a little redundant, but I think it's ok. More chance to catch any regressions.